apps/req.c

```diff
@@ -631,6 +636,11 @@ int req_main(int argc, char **argv)
             goto end;
         app_RAND_load_conf(req_conf, section);
     }
+    if (keyalg != NULL && pkey != NULL) {
+        BIO_printf(bio_err,
+                   "Warning: Not generating key via given -newkey option since -key is given\n");
+        /* Better throw an error in this case */
```

Comment:
Shouldn't we through it now?

Reply:
I don't really get what you mean with this comment here?
Please note that the addition, as done here, is already part of the `master` branch,
and I see little point in doing things differently in the `openssl-3.0` branch.

Reply:
OK, I agree. Sorry for disturbing.

Reply:
No worries.
Anyway, this warning (as well as other one that were proposed to be added here) are gone meanwhile for other reason.